backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5747500_sys_gh20159_Adjust_Docs_Invoice_Details_Footer.sql

```diff
@@ -0,0 +1,75 @@
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.Docs_Invoice_Details_Footer (IN p_Invoice_ID numeric,
+                                                                                        IN p_Language   Character Varying(6))
```

Comment:
delete this; was already fixed

Reply:
it's the same function i didn't change any thing, but when i checked in the UAT base it was not applied, so i'ts just a recreation script

Reply:
no need: was fixed already: https://github.com/metasfresh/metasfresh/commit/362aea5528240b6c6e88cb347c75ae9ccd519cfe